recipes/genomic_address_service/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - pip
```

Comment:
can you update the python constraint in the host section too?

```yaml
  - python >=3.10,<3.13
```

Reply:
Thanks so much. Done in [3feeda9](https://github.com/bioconda/bioconda-recipes/pull/54506/commits/3feeda9640ea92ffd6b681b701bac7d277896759)